bin/court_upsert/courts_data.py

```diff
@@ -0,0 +1,82 @@
+    468: {"name": "Harrow Crown Court", "type": "CROWN"},
+    469: {"name": "Wood Green Crown Court", "type": "CROWN"},
+    471: {"name": "Southwark Crown Court", "type": "CROWN"},
+    472: {"name": "Woolwich Crown Court", "type": "CROWN"},
+    475: {"name": "Isleworth Crown Court", "type": "CROWN"},
+    404: {"name": "Birmingham Crown Court & Annex", "type": "CROWN"},
+    417: {"name": "Coventry Combined Court Centre", "type": "CROWN"},
+    442: {"name": "Northampton Combined Court", "type": "CROWN"},
+    444: {"name": "Nottingham County Court & Family Court Crown Court", "type": "FAMILY"},
+    452: {"name": "Shrewsbury Crown Court", "type": "CROWN"},
+    455: {"name": "Stafford Combined Crown Court", "type": "CROWN"},
+    456: {"name": "Stoke-on-Trent Crown Court", "type": "CROWN"},
+    463: {"name": "Warwickshire (South) Justice Centre", "type": "CROWN"},
+    466: {"name": "Worcester Combined Court", "type": "CROWN"},
+    419: {"name": "Derby Combined Centre", "type": "CROWN"},
+    421: {"name": "Wolverhampton Crown Court", "type": "CROWN"},
+    430: {"name": "Leicester Crown Court", "type": "CROWN"},
+    432: {"name": "Lincoln Crown Court", "type": "CROWN"},
+    402: {"name": "Bradford Crown Court", "type": "CROWN"},
+    420: {"name": "Doncaster Crown Court (Doncaster Justice Centre South)", "type": "CROWN"},
+    422: {"name": "Durham Crown Court", "type": "CROWN"},
+    425: {"name": "Grimsby Combined Court Centre", "type": "CROWN"},
+    429: {"name": "Leeds Combined Court Centre", "type": "CROWN"},
+    439: {"name": "Newcastle Moot Hall Annex", "type": "CROWN"},
+    451: {"name": "Sheffield Combined Court Centre", "type": "CROWN"},
+    460: {"name": "Teesside Combined Court Centre", "type": "CROWN"},
+    467: {"name": "York Crown Court", "type": "CROWN"},
+    766: {"name": "Kingston-upon-Hull Combined Court Centre", "type": "CROWN"},
+    409: {"name": "Burnley Combined Court Centre", "type": "CROWN"},
+    412: {"name": "Carlisle Combined Court Centre", "type": "CROWN"},
+    415: {"name": "Chester Crown Court", "type": "CROWN"},
+    433: {"name": "Liverpool QEII Law Courts: Liverpool Crown Court", "type": "CROWN"},
+    435: {"name": "Manchester Crown Court (Crown Square)", "type": "CROWN"},
+    436: {"name": "Manchester Crown Court (Minshull Street)", "type": "CROWN"},
+    448: {"name": "Preston Combined Court Centre (Sessions House Annex)", "type": "CROWN"},
+    462: {"name": "Warrington Crown Court", "type": "CROWN"},
+    470: {"name": "Bolton Combined Court", "type": "CROWN"},
+    401: {"name": "Aylesbury Crown Court", "type": "CROWN"},
+    410: {"name": "Cambridge Crown Court", "type": "CROWN"},
+    414: {"name": "Chelmsford Crown Court", "type": "CROWN"},
+    426: {"name": "Ipswich Crown Court", "type": "CROWN"},
+    431: {"name": "Lewes Combined Court", "type": "CROWN"},
+    1602: {"name": "Hove Trial Centre", "type": "CROWN"},
+    431: {"name": "Brighton Magistrates Court", "type": "CROWN"},
```

Comment:
You have two different values for 431. Brighton court will overwrite Lewes court